### PR TITLE
Clarifications, more active voice, American spellings

### PR DESCRIPTION
**Pull request by Technicalleigh**
> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in [How to contribute](https://github.com/thegooddocsproject/templates/blob/master/contribute.md#contributing).__

## Purpose / why

> _**Replace this text** - Restate the purpose/justification of this work and include links to the Issue inside of the purpose/why._

## What changes were made?

> _**Replace this text** - State clearly the direct additions or modifications made in this change-request._

## Verification

> _**Replace this text** - Document the details that help a reviewer understand what steps or actions should be taken in-order to verify the work is completed and satisfies the initial scope of work._

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?

### CODE REVIEW

File: how-to/about-how-to.md [hunk 1]
```diff
@@ -5,35 +5,35 @@
...
+New users who want to solve a problem immediately can also benefit from how-to articles, provided they are well-written and clearly state any prerequisite knowledge required.
 
 ## Content of your how-to article
 
 ### About the "Overview" section
 
 Summarize what the reader will get from reading the how-to articles.
 
-Creating a good overview comes down to thinking about who is going to be using it:
+Creating a good overview comes down to thinking about will use the doc:
```
Comment (Loquacity) [inline how-to/about-how-to.md:22]:
```suggestion
Creating a good overview comes down to thinking about who will be reading the document:
```

File: how-to/about-how-to.md [hunk 2]
```diff
@@ -5,35 +5,35 @@
...
+This section's purpose is to prevent that from happening.
 
-Prerequisites can include other articles or information that needs to be read, or it can be technical dependencies such as requiring an internet connection.
-Describe what the audience needs to know, or needs to have, before attempting this how-to.
-By stating the requirements up-front, you prevent your audience from having a bad experience with your tutorial.
-You can include links to procedures or information, or give your audience useful pointers about how to get whatever they need.
+Describe what the audience needs to know or have before following this how-to. Prerequisites can include other articles or information
+the reader needs to understand first. This list can also include technical dependencies, such as requiring an internet connection.
+By stating the requirements up-front, you prevent your audience from having a bad experience with your how-to.
+Whenever possible, include links to procedures or information, or give your audience tips about how to get whatever they need.
```
Comment (Loquacity) [inline how-to/about-how-to.md:36]:
```suggestion
Describe what the reader needs to know or have before following this how-to. Prerequisites can include other articles or information the reader needs to understand first. This list can also include technical dependencies, such as requiring an internet connection. By stating the requirements up-front, you prevent your audience from having a bad experience with your how-to. Whenever possible, include links to procedures or information, or give your audience tips about how to get whatever they need.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: how-to/about-how-to.md (hunk 2 above)
Comment (Loquacity) [reply how-to/about-how-to.md:36]: Not entirely certain what's happening with the line breaks here.

File: how-to/about-how-to.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Make sure you meet the following prerequisites before following the steps:
 * Access to the Postman application.
 * An understanding about what a RESTful API is conceptually. For more information about RESTful APIs, see http://example.com/restful_apis. 
 * (Optional) A development tool (IDE) that displays API responses formatted for readability.
-* You need to have a list of favourites before you can manage them. For more information about favourites lists, see #Create_Favourites.
+* You need to have a list of favorites before you can manage them. For more information about favourites lists, see #Create_Favorites.
```
Comment (aidanatwork) [inline how-to/about-how-to.md:46]: We still need to change to the American spelling for "favorites" in the last instance at the end of this sentence.

**Comment by Loquacity** [comment]
This has picked up a bunch of conflicts with my PR#96 Would you like to me go through and fix the conflicts, or should we close this PR without merging?

**Comment by aidanatwork** [comment]
> This has picked up a bunch of conflicts with my PR #162 Would you like to me go through and fix the conflicts, or should we close this PR without merging?

@Loquacity Please go ahead and try to resolve the conflicts. If you are getting stuck, feel free to reach out to me or Morgan Craft to help further.

**Comment by Loquacity** [comment]
Merge conflicts resolved. Ready to merge once others are happy.

**Review by Loquacity — APPROVED** [review]

**Pull request merged**